internal/ui/graphics.go

```diff
@@ -39,13 +39,14 @@ func newGraphicsDriver(creator graphicsDriverCreator) (graphicsdriver.Graphics,
 
 	switch env {
 	case "", "auto":
-		g, err := creator.newAuto()
+		g, lib, err := creator.newAuto()
 		if err != nil {
 			return nil, err
 		}
 		if g == nil {
 			return nil, fmt.Errorf("ui: no graphics library is available")
 		}
+		theGlobalState.graphicsLibrary = lib
```

Comment:
Create methods to set/get the graphics library and use atomic.StoreInt32 and LoadInt32 in the implementations.